resources/user.go

```diff
@@ -39,11 +42,13 @@ const passwdFile = "/etc/passwd"
 // UserRes is a user account resource.
 type UserRes struct {
 	BaseRes           `yaml:",inline"`
-	State             string  `yaml:"state"` // state: exists, absent
-	UID               *uint32 `yaml:"uid"`
-	GID               *uint32 `yaml:"gid"`
-	HomeDir           *string `yaml:"homedir"`
-	AllowDuplicateUID bool    `yaml:"allowduplicateuid"`
+	State             string   `yaml:"state"` // state: exists, absent
+	UID               *uint32  `yaml:"uid"`
```

Comment:
sorry about this, I should have mentioned it earlier, but can you add some short docs as comment at the end of some of the parameters? if you think it's important to have a long doc string, then it can be put above the field in full sentence form. These show up in the docs.